Bugzilla/WebService/User.pm

```diff
@@ -135,6 +137,61 @@ sub create {
     return { id => $self->type('int', $user->id) };
 }
 
+sub suggest {
+    my ($self, $params) = @_;
+
+    Bugzilla->switch_to_shadow_db();
+
+    ThrowCodeError('params_required', { function => 'User.suggest_users', params => ['match'] })
+      unless defined $params->{match};
+
+    ThrowUserError('user_access_by_match_denied')
+      unless Bugzilla->user->id;
+
+    untaint($params->{match});
+    my $s = $params->{match};
+    trim($s);
+
+    my $dbh = Bugzilla->dbh;
+    my @select = ('realname AS real_name', 'login_name AS name');
+    my $order  = 'last_seen_date DESC';
+    my $where;
+    state $have_mysql = $dbh->isa('Bugzilla::DB::Mysql');
```

Comment:
it's a bit weird to have db-specific paths outside of the `Bugzilla::DB` / `Bugzilla::DB::Mysql` classing

Reply:
I'm going to accept this weirdness for now, because I don't feel like looking up the postgres version just yet (but I'd rather not add to the non-mysql sql that someone will need to fix in harmony)